app/cli/internal/action/attestation_push.go

```diff
@@ -224,6 +232,50 @@ func (action *AttestationPush) Run(ctx context.Context, attestationID string, ru
 	return attestationResult, nil
 }
 
+func (action *AttestationPush) saveBundle(bundle *protobundle.Bundle) error {
+	// Save bundle to disk if requested
+	if action.bundlePath != "" {
+		bundleJSON, err := fixBundle(bundle)
+		if err != nil {
+			return fmt.Errorf("encoding bundle: %w", err)
+		}
+		action.Logger.Info().Msg(fmt.Sprintf("Storing Sigstore bundle %s", action.bundlePath))
+		err = os.WriteFile(action.bundlePath, bundleJSON, 0600)
+		if err != nil {
+			return fmt.Errorf("writing bundle: %w", err)
+		}
+	}
+
+	return nil
+}
+
+func fixBundle(bundle *protobundle.Bundle) ([]byte, error) {
+	env := bundle.GetDsseEnvelope()
+	sig := env.GetSignatures()[0].GetSig()
```

Comment:
It's unluckily but, I would check there are signatures here

Reply:
Yep, that case would be weird, since there are no envelopes without signatures. I'll add the check in any case. Thanks.